components/layout/display_list/builder.rs

```diff
@@ -1375,104 +1323,7 @@ impl Fragment {
-                bounds: stacking_relative_border_box.to_layout(),
-                common: items::empty_common_item_properties(),
-                widths: SideOffsets2D::new_all_same(Au::from_px(1)).to_layout(),
-                details: BorderDetails::Normal(border::simple(
-                    ColorU::new(0, 0, 200, 1).into(),
-                    webrender_api::BorderStyle::Solid,
-                )),
-            },
-            Vec::new(),
-        )));
-
-        // Draw a rectangle representing the baselines.
-        let mut baseline = LogicalRect::from_physical(
-            self.style.writing_mode,
-            stacking_relative_content_box,
-            container_size,
-        );
-        baseline.start.b += text_fragment.run.ascent();
-        baseline.size.block = Au(0);
-        let baseline = baseline.to_physical(self.style.writing_mode, container_size);
-
-        let base = state.create_base_display_item(
-            clip,
-            self.node,
-            self.unique_id(),
-            get_cursor(style, Cursor::Default),
-            DisplayListSection::Content,
-        );
-        // TODO(gw): Use a better estimate for wavy line thickness.
-        let area = baseline.to_layout();
-        let wavy_line_thickness = (0.33 * area.size().height).ceil();
-        state.add_display_item(DisplayItem::Line(CommonDisplayItem::new(
-            base,
-            webrender_api::LineDisplayItem {
-                common: items::empty_common_item_properties(),
-                area,
-                orientation: webrender_api::LineOrientation::Horizontal,
-                wavy_line_thickness,
-                color: ColorU::new(0, 200, 0, 1).into(),
-                style: LineStyle::Dashed,
-            },
-        )));
-    }
-
-    /// Adds display items necessary to draw debug boxes around this fragment.
-    fn build_debug_borders_around_fragment(
-        &self,
-        state: &mut DisplayListBuildState,
-        stacking_relative_border_box: Rect<Au>,
-        clip: Rect<Au>,
-    ) {
-        // This prints a debug border around the border of this fragment.
-        let base = state.create_base_display_item(
-            clip,
-            self.node,
-            self.unique_id(),
-            get_cursor(&self.style, Cursor::Default),
-            DisplayListSection::Content,
-        );
-        state.add_display_item(DisplayItem::Border(CommonDisplayItem::with_data(
-            base,
-            webrender_api::BorderDisplayItem {
-                bounds: stacking_relative_border_box.to_layout(),
-                common: items::empty_common_item_properties(),
-                widths: SideOffsets2D::new_all_same(Au::from_px(1)).to_layout(),
-                details: BorderDetails::Normal(border::simple(
-                    ColorU::new(0, 0, 200, 1).into(),
-                    webrender_api::BorderStyle::Solid,
-                )),
-            },
-            Vec::new(),
-        )));
-    }
-
+    /// Adds display
```

Comment:
This seems a leftover from the removal

Reply:
Thanks for pointing this out. I'll remove it.